Makefile

```
@@ -28,13 +28,21 @@ help: ## Display this help
 		}\
 	} BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9%/_-]+:.*?##/ { printf "  \033[36m%-18s\033[0m %s\n", "make " $$1, ww($$2) } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
+# The go.work file is git ignored and not checked in. Create one if
+# it doesn't exist already. (Note we're using absolute paths.)
+go.work:
+	ROOT=$$(git rev-parse --show-toplevel) && \
+		cd $${ROOT} && \
+		go work init && \
+		go work use -r $${ROOT}
+
```

It we assume we're running from the root directory it could be shorter...

```suggestion
go.work:
	go work init && go work use -r $${PWD}
```

It's probably safe to assume that, but I decided to go for the extra strength robustness. :shrug: .

---

Another option if we don't care about the absolute path:

```suggestion
go.work:
	go work init && go work use -r .
```